website/_data/docs.yml

```diff
@@ -78,6 +77,10 @@
   - howto/howto_use_cli
   - howto/howto_ldap_and_sso
   - howto/howto_use_beeline
-  - howto/howto_update_coprocessor
+  - howto/howto_upgrade_coprocessor
```

Comment:
This change will change the URL endpoint. We hope the URL can be kept as there are many external references.